bot/bot.py

```diff
@@ -167,14 +167,14 @@ async def fake_gen():
 
                 answer = answer[:4096]  # telegram message limit
                 if i == 0:  # send first message (then it'll be edited if message streaming is enabled)
-                    if len(answer) == 0:  # first answer chunk from openai was empty
-                        i = -1  # try again to send first message
-                        continue
-
                     try:                    
                         sent_message = await update.message.reply_text(answer, parse_mode=parse_mode)
-                    except telegram.error.BadRequest:
-                        sent_message = await update.message.reply_text(answer)
+                    except telegram.error.BadRequest as e:
+                        if str(e).startswith("Message must be non-empty"):  # first answer chunk from openai was empty
```

Comment:
Should we use regex `r"^\s*$"` for robustness?
```python
import re
if re.match(r"^\s*$", answer):
    continue
```

Reply:
We just use Telegram to check if message is empty. It's the most valid check

Reply:
Ok, make sense.